docs/resource-hsaccess.txt

```diff
@@ -0,0 +1,35 @@
+/home/igray/hydroshare/hs_modelinstance/models.py
```

Comment:
@ilangray after looking into docs/ folder, I see other screen capture images showing some docker failures are also included. I personally think this docs/ folder needs to be cleared up to only include generic documentations since it is included as part of the code repo. Since this is not the only exception, I will merge this PR now, and note this issue here. If others agree this doc folder needs to be cleared up, an issue may be created so it can be addressed as a whole.